java-sdk/nav-hinting-integration-test.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+echo
+echo "Usage: contextual-bandit-integration-test.sh run from harness/java-sdk without SSL or Auth"
+echo
+# point to the harness host, use https://... for SSL and set the credentials if using Auth
+# export "HARNESS_CLIENT_USER_ID"=xyz
+# export "HARNESS_CLIENT_USER_SECRET"=abc
+host=localhost
+engine=hinting
+test_queries=data/nh-queries.json
+user1_events=data/nh-pferrel-events.json
+user2_events=data/nh-joe-events.json
+user1_conversion=data/nh-pferrel-conversion.json
+user2_conversion=data/nh-joe-conversion.json
+sleep_seconds=1
+
+export HARNESS_SERVER_CERT_PATH=/Users/pat/harness/rest-server/server/src/main/resources/keys/harness.pem
```

Comment:
Should this be in the script? I don't think so. The test should run with Harness as it is configured, requiring no config in the script.